lib/phone_shop/widgets/components/shop_gallery.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+class ShopGallery extends StatelessWidget {
+  final List<String> images;
+  final EdgeInsetsGeometry padding;
+
+  const ShopGallery({super.key, required this.images, required this.padding});
+
+  @override
+  Widget build(BuildContext context) {
+    Widget productListView;
+
+    productListView = ListView.builder(
+      scrollDirection: Axis.horizontal,
+      itemCount: images.length,
+      itemBuilder: (context, index) {
+        return Padding(
+          padding: padding,
+          child: _ImageItem(image: images[index]),
+        );
```

Comment:
Do not use this way of separating Images inside Gallery, and there is better way to add padding on the beginning and end of list, and there is way how to separate images inside list